### DISCUSSION TITLE
能不能出一个JavaScript的版本，typescript的我安装后各种报错

### DISCUSSION THREAD

**Discussion by yangfan5157** [ant-design/pro-components#2548] [Ideas]
3ks

**Comment by yingzhi0808** [comment]
打包后的文件不就是 JS 的吗

**Comment by yangfan5157** [comment]
我下载下来源码看是typeScript的




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ant-design/pro-components"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月20日(星期二) 下午5:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ant-design/pro-components] 能不能出一个JavaScript的版本，typescript的我安装后各种报错 (#2548)





 
打包后的文件不就是 JS 的吗

**Reply by yangfan5157** [reply]
./node_modules/@ant-design/pro-form/es/layouts/DrawerForm/index.js
Module not found: Can't resolve 'rc-util/es/Dom/scrollLocker' in 'D:\work\开发测试\eureka\node_modules\@ant-design\pro-form\es\layouts\DrawerForm'
我用命令安装的，发现各种错误，编译一直不通过，不知道是版本的问题吗？我是通过npm i重新安装的都不行

**Comment by aiyudeleng** [comment]
请问楼主找到了吗。。。
